server.js

```diff
@@ -5,16 +5,16 @@ var fs         = require('fs');
 var SNSClient  = require('aws-snsclient');
 var winston    = require('winston');
 
-winston.add(winston.transports.File, { filename: '/app/log/ses_json.log' });
-winston.remove(winston.transports.Console);
+const logFile = new winston.transports.File({ filename: './log/ses_json.log' });
+winston.add(logFile);
 
 require('console-stamp')(console, '[yyyy-mm-dd HH:MM:ss.l Z]');
 
-var db_config = yaml.safeLoad(fs.readFileSync('/app/database.yml', 'utf8'));
-var auth = yaml.safeLoad(fs.readFileSync('/app/auth.yml', 'utf8'));
+var db_config = yaml.load(fs.readFileSync('./database.yml', 'utf8'));
+var auth = yaml.load(fs.readFileSync('./auth.yml', 'utf8'));
```

Comment:
[`yaml.load`](https://github.com/nodeca/js-yaml#load-string---options-) is now safe by default.